releasenotes/notes/fix_loop-35cd0efcf5a0d0b6.yaml

```
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Fixed bug where set elements were removed inside a loop that iterates
+    over the set elements.In addition, if a gate is defined in the supported instructions
+    but not in the 'properties', skip that gate.
```

```suggestion
    A bug that the backend target generation fails with an error 
    in some edge cases is fixed. See Qiskit/qiskit-ibm-provider/#650
    for more details.
```
I think this is too much details for end-users.  No one will understand this note without reading the code.